src/java_binding/Cargo.toml

```diff
@@ -17,6 +17,7 @@ jni = "0.21.1"
 prost = "0.11"
 risingwave_common = { workspace = true }
 risingwave_hummock_sdk = { workspace = true }
+risingwave_jni_core = { workspace = true }
```

Comment:
I guess here we only depends on `risingwave_jni_core`, and we can remove the rest.